buildSrc/build.gradle.kts

```diff
@@ -7,7 +7,7 @@ repositories {
 }
 
 dependencies {
-    implementation("com.android.tools.build:gradle:7.4.2")
+    implementation("com.android.tools.build:gradle:8.2.0")
     implementation("org.jetbrains.kotlin:kotlin-gradle-plugin:1.9.20")
-    implementation("org.jetbrains.kotlin:kotlin-stdlib:1.8.22")
+    implementation("org.jetbrains.kotlin:kotlin-stdlib:1.9.21")
```

Comment:
A Kotlin standard library 1.9.21 update comes along with the update to Gradle 8.2 and Android Gradle Plugin 8.2.0.  If I recall the testing correctly, this aids downstream in avoiding the template repository's issues in using the new Android core-ktx 1.12.0 version.